scripts/push_pull_run_element_info.py

```diff
@@ -61,6 +63,10 @@ def _run(self):
 
 class PushRunElementInfo(StepEPP):
 
+    @cached_property
+    def now(self):
+        return datetime.datetime.now()
+
```

Comment:
This should work fine as is, but the name is potentially misleading - if we call it a second time, we will get the cached time, not the current one.